app/components/attachment_button.js

```diff
@@ -65,12 +67,93 @@ export default class AttachmentButton extends PureComponent {
         intl: intlShape.isRequired,
     };
 
+    getPermissionDeniedMessage = (source, mediaType = '') => {
+        const {formatMessage} = this.context.intl;
+        const applicationName = DeviceInfo.getApplicationName();
+        switch (source) {
+        case 'camera': {
+            if (mediaType === 'video') {
+                return {
+                    title: formatMessage({
+                        id: 'mobile.camera_video_permission_denied_title',
+                        defaultMessage: 'Camera access is required',
```

Comment:
default message does not match en.json

Reply:
Changed all default message to match with en.json